src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -381,6 +381,17 @@ ensogl::define_endpoints! {
         toggle_fullscreen_for_selected_visualization(),
 
 
+        // === Scene Navigation ===
+
+
+        /// Stop the scene camera from moving around, locking the scene in place.
+        /// Can be used, e.g., if there is a fullscreen visualisation active, or navigation should
+        ///only work for a selected visualisation.
+        disable_navigator(),
+        /// Allow the scene camera to move around for navigating the scene.
+        enable_navigator(),
```

Comment:
Over the last months I have learned that its always better to keep that as a single endpoint (`set_navigator_active(bool)`) rather than two, unless you really want to connect it to keyboard shortcuts (which we do not want). So lets use single endpoint if possible.